proof/crefine/Move_C.thy

```diff
@@ -1480,19 +1480,6 @@ lemma multiple_add_less_nat:
   apply simp
   done
 
-(* FIXME AARCH64 move to wherever we keep the from_bool/to_bool lemmas *)
-lemma word_le_1_and_idem:
```

Comment:
can you confirm that Word_Lemmas_Internal are where we keep our from_bool/to_bool lemmas?

Reply:
Word_Lemmas_Internal is the area to commit Word_Lib lemmas to that should be moved into the rest of Word_Lib at the next sync with the AFP. The from_bool/to_bool lemmas are deeper within Word_Lib, but should only move there at the sync point (otherwise we risk accidentally removing things when we sync).

Reply:
Oki, but then do you want to remove or keep the comment to move it near from/to_bool?

Reply:
`word_le_1_and_idem` should not actually go there, `from_to_bool_le_1_idem` should. `word_le_1_and_idem` should go where `AND` lemmas are.